bulk.go

```diff
@@ -10,7 +10,7 @@ import (
 	"fmt"
 	"net/url"
 
-	"golang.org/x/net/context"
+	"context"
```

Comment:
Can you please group the imports into stdlib / external / internal imports. Tools like `goimports` should help you with that:

Instead of ...

```
import (
	"bytes"
	"errors"
	"fmt"
	"net/url"

	"context"

	"gopkg.in/olivere/elastic.v3/uritemplates"
)
```

... it should be ...

```
import (
	"bytes"
	"context"
	"errors"
	"fmt"
	"net/url"

	"gopkg.in/olivere/elastic.v3/uritemplates"
)
```

Reply:
Thanks for reviewing. Seems like `goimports ` won't help for those kind imports. Actually, I did run `goimports` before I commit. Now I group them manually. Could you please review again?

Reply:
With `-local`, it should [group the imports](https://stackoverflow.com/questions/38704769/golang-import-grouping-by-package), no?